hub-js/src/local/query/spec-value-field.ts

```diff
@@ -0,0 +1,113 @@
+import { logger } from "../../logger";
```

Comment:
**Question to reviewer**
Where this file should be located? It's both `mutation` and `query` resolver as it's defined on field.
Should I change the structure to sth like:
```bash
└── local
   ├── resolver
   │  ├── mutation # here current mutation resolvers
   │  └── field  # here new `value` field resolver
   └── storage

```